public/scripts/menu.js

```diff
@@ -0,0 +1,86 @@
+miniDom = {};
+
+miniDom.hasClass = function(el, className){
+    if (el.classList)
+        return el.classList.contains(className);
+    else
+        return new RegExp('(^| )' + className + '( |$)', 'gi').test(el.className);
+}
+
+miniDom.removeClass = function(el, className){
+    if (el.classList)
+      el.classList.remove(className);
+    else
+      el.className = el.className.replace(new RegExp('(^|\\b)' + className.split(' ').join('|') + '(\\b|$)', 'gi'), ' ');
+}
+
+miniDom.matches = function(el, selector) {
+    // shortcut to use matches globally
+    if(typeof selector === "undefined"){
+        selector = el;
+        el = document.body;
+    }
+    return (el.matches || el.matchesSelector || el.msMatchesSelector || el.mozMatchesSelector || el.webkitMatchesSelector || el.oMatchesSelector).call(el, selector);
+};
```

Comment:
Should only need to look for `msMatchesSelector`.
http://caniuse.com/#feat=matchesselector